pipelines/folioRancher/folioDevTools/moduleDeployment/deploySidecarFromFeatureBranch/Jenkinsfile

```diff
@@ -0,0 +1,157 @@
+package folioRancher.folioDevTools.moduleDeployment.deploySidecarFromFeatureBranch
+
+import org.folio.Constants
+import org.folio.models.EurekaNamespace
+import org.folio.utilities.Logger
+import org.folio.utilities.GitHubClient
+import org.jenkinsci.plugins.workflow.libs.Library
+
+@Library('pipelines-shared-library@RANCHER-1894') _
+
+/** Pipeline properties and parameters */
+properties([
+  buildDiscarder(logRotator(numToKeepStr: '30')),
+  disableConcurrentBuilds(),
+  parameters([
+    string(name: 'MODULE_NAME', defaultValue: 'folio-module-sidecar', description: 'GitHub Repository Name for Sidecar application'),
```

Comment:
If it is espesially job for sidecar only, we could remove this parameter

Reply:
@OHaimanov 

What if Sidecar name is changed some day or we switch to another one?